test/requirements.yml

```diff
@@ -1,9 +0,0 @@
-dependencies:
```

Comment:
This file was introduced to run test when we finished conda release. But, now we only do smoke test after release. So this file is not used anywhere in the code base.